packages/altive_lints/lib/src/lints/avoid_single_child.dart

```diff
@@ -6,7 +6,7 @@ import 'package:custom_lint_builder/custom_lint_builder.dart';
 /// An `avoid_single_child` rule that warns against using layout
 /// widgets intended for multiple children with only one child.
 ///
-/// This includes widgets like `Column`,`Row`, `Flex`, `Wrap`,
+/// This includes widgets like `Column`,`Row`, `Flex`, `Wrap`,`Stack`
```

Comment:
![badge](https://img.shields.io/badge/review-nits-inactive)
Align comma and space conventions.
```suggestion
/// This includes widgets like `Column`, `Row`, `Stack`, `Flex`, `Wrap`
```